### PR TITLE
MKE Virtualization (KubeVirt) section

### PR DESCRIPTION
**Pull request by KoryKessel-Mirantis**

### CODE REVIEW

**Comment by byDimasik** [comment]
@sergeymirantis 

> @KoryKessel-Mirantis - who is the approver now? Why can't I approve? @byDimasik do you know?

Try that again, I've updated your permissions

**Review by sergeymirantis — APPROVED** [review]
looks good - please move on.

File: content/docs/operations/mke-virtualization/_index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: MKE Virtualization
+weight: 1
+---
+
+Virtualization functionality is available for MKE through [KubeVirt](https://kubevirt.io/), a Kubernetes extension with which you can natively run
+Virtual Machine (VM) workloads alongside container workloads in Kubernetes
```
Comment (ogusarenko) [inline content/docs/operations/mke-virtualization/_index.md:7]: just -- virtual machine

File: content/docs/operations/mke-virtualization/_index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+clusters.
```
Comment (ogusarenko) [inline content/docs/operations/mke-virtualization/_index.md:8]: I understand that it works this way as well, but let's keep the sources nice and well-formatted. Move the word "clusters" to the previous line, there is enough space there.

File: content/docs/operations/mke-virtualization/install-virtctl-cli.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+title: Install virtctl CLI
+weight: 3
+---
+
+Although it is not required to run KubeVirt, the virtctl CLI provides an
```
Comment (ogusarenko) [inline content/docs/operations/mke-virtualization/install-virtctl-cli.md:6]: mark "virtctl" as a command/tool

File: content/docs/operations/mke-virtualization/install-virtctl-cli.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+interface that can significantly enhance the convenience of your virtual
+machine interactions.
+
+1. Obtain the virtctl version for your particular architecture from
+   https://binary.mirantis.com/?prefix=kubevirt/bin/artifacts.
+
+2. Run the following command, inserting the correct values for your
+   architecture and platform. For **ARCH** the valid values are
```
Comment (ogusarenko) [inline content/docs/operations/mke-virtualization/install-virtctl-cli.md:14]: use backticks to format the inline text as code

File: content/docs/operations/mke-virtualization/install-virtctl-cli.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+   wget https://binary-mirantis-com.s3.amazonaws.com/kubevirt/bin/artifacts/virtctl-1.3.1-20240911005512-linux-amd64 -O virtctl
+   ```
+
+   Example command for MacOS with arm64 architecture:
+
+   ```bash
+   wget https://binary-mirantis-com.s3.amazonaws.com/kubevirt/bin/artifacts/virtctl-1.3.1-20240911005512-darwin-arm64  -O virtctl
+   ```
+
+3. Move virtctl to one of the :guilabel:`PATH` directories.
```
Comment (ogusarenko) [inline content/docs/operations/mke-virtualization/install-virtctl-cli.md:34]: guilabel is not working in md

File: content/docs/operations/mke-virtualization/install-virtctl-cli.md (hunk 4 above)
Comment (ogusarenko) [reply content/docs/operations/mke-virtualization/install-virtctl-cli.md:14]: check throughout the commit

File: content/docs/operations/mke-virtualization/virtualization-use-scenario.md [hunk 6]
```diff
@@ -0,0 +1,248 @@
+---
+title: MKE virtualization deployment scenario
+weight: 5
+---
+
+The example scenario illustrated herein pertains to the deployment of a CirrOS
+virtual machine. The scenario consists of three primary steps:
```
Comment (ogusarenko) [inline content/docs/operations/mke-virtualization/virtualization-use-scenario.md:7]: Remove the exact number of steps from "The scenario consists of three primary steps:"
This way the content is easier to maintain

File: content/docs/operations/mke-virtualization/virtualization-use-scenario.md [hunk 7]
```diff
@@ -0,0 +1,248 @@
...
+   spec:
+     storageClassName: hpp-local
+     accessModes:
+       - ReadWriteOnce
+     resources:
+       requests:
+         storage: 3Gi
+   ```
+
+2. Run the following command to create the `PersistentVolumeClaim` resource
```
Comment (ogusarenko) [inline content/docs/operations/mke-virtualization/virtualization-use-scenario.md:120]: Create the `PersistentVolumeClaim` resource that you defined in the `example-pvc.yaml` file:

File: content/docs/operations/mke-virtualization/virtualization-use-scenario.md [hunk 8]
```diff
@@ -0,0 +1,248 @@
...
+   ```
+
+   Example output:
+
+   ```bash
+   NAME          STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS    AGE
+   pvc-claim-1   Bound    pvc-b7d68902-f340-4b7e-8a36-495d170b7193   10Gi       RWO            hpp-local       10s
+   ```
+
+4. Run the following command to attach the disk to your VM:
```
Comment (ogusarenko) [inline content/docs/operations/mke-virtualization/virtualization-use-scenario.md:140]: 4. Attach the disk to your virtual machine:

File: content/docs/operations/mke-virtualization/virtualization-use-scenario.md [hunk 9]
```diff
@@ -0,0 +1,248 @@
...
+
+   ```bash
+   kubectl patch vm vm-cirros --type='json' -p
+   '[{"op":"add","path":"/spec/template/spec/volumes/2","value":{"persistentVolumeClaim":
+   {"claimName": "pvc-claim-1"},"name":
+   "example-pvc"}},{"op":"add","path":"/spec/template/spec/domain/devices/disks/2","value":{"disk":
+   {"bus": "virtio"},"name": "example-pvc"}}]'
+   ```
+
+5. Restart the VM and access the console:
```
Comment (ogusarenko) [inline content/docs/operations/mke-virtualization/virtualization-use-scenario.md:150]: VM - virtual machine
you use it differently through the content, be consitent

File: content/docs/operations/mke-virtualization/virtualization-use-scenario.md [hunk 10]
```diff
@@ -0,0 +1,248 @@
...
+       }'
+   ```
+
+2. Apply the `bridge-test.yaml` file:
+
+   ```bash
+   kubectl apply -f bridge-test.yaml
+   ```
+
+3. Run the following command to attach the network interface to your virtual
```
Comment (ogusarenko) [inline content/docs/operations/mke-virtualization/virtualization-use-scenario.md:209]: 3. Attach the network interface to your virtual machine:

File: content/docs/operations/mke-virtualization/_index.md (hunk 1 above)
Comment (KoryKessel-Mirantis) [reply content/docs/operations/mke-virtualization/_index.md:7]: Done.

File: content/docs/operations/mke-virtualization/_index.md (hunk 2 above)
Comment (KoryKessel-Mirantis) [reply content/docs/operations/mke-virtualization/_index.md:8]: Done.

File: content/docs/operations/mke-virtualization/install-virtctl-cli.md (hunk 3 above)
Comment (KoryKessel-Mirantis) [reply content/docs/operations/mke-virtualization/install-virtctl-cli.md:6]: Done.

File: content/docs/operations/mke-virtualization/install-virtctl-cli.md (hunk 4 above)
Comment (KoryKessel-Mirantis) [reply content/docs/operations/mke-virtualization/install-virtctl-cli.md:14]: Done.

File: content/docs/operations/mke-virtualization/virtualization-use-scenario.md (hunk 6 above)
Comment (KoryKessel-Mirantis) [reply content/docs/operations/mke-virtualization/virtualization-use-scenario.md:7]: Edited to read:

"The example scenario illustrated herein pertains to the deployment of a CirrOS virtual machine, which is comprised by the following primary steps:"

File: content/docs/operations/mke-virtualization/virtualization-use-scenario.md (hunk 7 above)
Comment (KoryKessel-Mirantis) [reply content/docs/operations/mke-virtualization/virtualization-use-scenario.md:120]: Done.

File: content/docs/operations/mke-virtualization/virtualization-use-scenario.md (hunk 8 above)
Comment (KoryKessel-Mirantis) [reply content/docs/operations/mke-virtualization/virtualization-use-scenario.md:140]: Done.

File: content/docs/operations/mke-virtualization/virtualization-use-scenario.md (hunk 9 above)
Comment (KoryKessel-Mirantis) [reply content/docs/operations/mke-virtualization/virtualization-use-scenario.md:150]: As the audience refers to flips so easily between "virtual machine" and "VM" I made a conscious choice to do the same. Still, I know you are correct...even if you do have trouble spelling 'consistent'. :-)

Done.

File: content/docs/operations/mke-virtualization/virtualization-use-scenario.md (hunk 10 above)
Comment (KoryKessel-Mirantis) [reply content/docs/operations/mke-virtualization/virtualization-use-scenario.md:209]: Done.

**Review by ogusarenko — APPROVED** [review]

**Pull request merged**